src/main/java/intellistart/interviewplanning/model/candidateslot/CandidateSlotService.kt

```diff
@@ -1,15 +1,21 @@
 package intellistart.interviewplanning.model.candidateslot
 
+import intellistart.interviewplanning.bean.post.processor.candidate.slot.validation.ValidatingCandidateSlot
+import intellistart.interviewplanning.bean.post.processor.candidate.slot.validation.ValidationMethod
 import intellistart.interviewplanning.exceptions.SlotException
 import intellistart.interviewplanning.exceptions.SlotException.SlotExceptionProfile
+import org.springframework.beans.factory.annotation.Autowired
 import java.time.LocalDate
 import org.springframework.stereotype.Service
 
 /**
  * Service for CandidateSlot entity.
  */
 @Service
-class CandidateSlotService(private val candidateSlotRepository: CandidateSlotRepository) {
+class CandidateSlotService{
+
+    @Autowired
+    private lateinit var candidateSlotRepository: CandidateSlotRepository
```

Comment:
why did you switched to field injection?

Reply:
I'm still using maven and the plugin without arguments didn't work, but I've already fixed that. You'll see in the next commit